calculator/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <title>Calculator App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="calc.css">
+    <link rel="icon" href="img/redi.png">
+</head>
+<body>
+<div class="headerBar">
+    <ul class="headerBar__links">
+        <li><a class="headerBar__link" href="/">Home</a></li>
+        <li><a class="headerBar__link headerBar__link--active" href="#">Calculator</a></li>
+    </ul>
+</div>
+
+<div class="mainArea mainArea--calc">
```

Comment:
Input box is not moved down when content is growing (even though div is growing)
![image](https://user-images.githubusercontent.com/19416132/79195427-8a2cab00-7e2e-11ea-9ce9-da764d876551.png)

Reply:
Yeah its a quick demo calculator. I think it is close enough. It is not listed as a use case to that it has to to handle more then a few numbers.

Reply:
Updated colors to match REDI theme